decision_service/dsclientlib/ds_config_collection.cc

```diff
@@ -0,0 +1,22 @@
+#include "ds_config_collection.h"
+
+namespace decision_service { namespace utility {
+
```

Comment:
I'm a little bit unclear on the utility of this class.  If you discard the class, then you don't need to even define set since _map[name]=value is fine.  Defining two helper get_default() in a namespace seems plenty straightforward.

Reply:
Would not be needed right now if we are fine with exposing stl in our interface.
In the long run, I was thinking about this as a message that can be sent to microservices that will allow us to change behavior at runtime.  For example it would be nice to change the log level at runtime.  

---
In reply to: [186775278](https://github.com/JohnLangford/vowpal_wabbit/pull/1479#discussion_r186775278) [](ancestors = 186775278)

Reply:
Ok.